docs/source/conf.py

```diff
@@ -54,15 +54,15 @@
 
 # -- Project documents -------------------------------------------------------
 # export the READme
-with open(os.path.join(PATH_ROOT, 'README.md'), 'r') as fp:
-    readme = fp.read()
-# TODO: temp fix removing SVG badges and GIF, because PDF cannot show them
-readme = re.sub(r'(\[!\[.*\))', '', readme)
-readme = re.sub(r'(!\[.*.gif\))', '', readme)
-for dir_name in (os.path.basename(p) for p in glob.glob(os.path.join(PATH_ROOT, '*')) if os.path.isdir(p)):
-    readme = readme.replace('](%s/' % dir_name, '](%s/%s/' % (PATH_ROOT, dir_name))
-with open('readme.md', 'w') as fp:
-    fp.write(readme)
+# with open(os.path.join(PATH_ROOT, 'README.md'), 'r') as fp:
+#     readme = fp.read()
+# # TODO: temp fix removing SVG badges and GIF, because PDF cannot show them
+# readme = re.sub(r'(\[!\[.*\))', '', readme)
+# readme = re.sub(r'(!\[.*.gif\))', '', readme)
+# for dir_name in (os.path.basename(p) for p in glob.glob(os.path.join(PATH_ROOT, '*')) if os.path.isdir(p)):
+#     readme = readme.replace('](%s/' % dir_name, '](%s/%s/' % (PATH_ROOT, dir_name))
+# with open('readme.md', 'w') as fp:
+#     fp.write(readme)
```

Comment:
Commented out these lines instead of deleting them following the (strange?) convention in PL:
https://github.com/PyTorchLightning/pytorch-lightning/blob/a053d758d03558d2aa5a328b2f6befbc133a0ebc/docs/source/conf.py#L43-L51